tests/src/rules/no-cycle.js

```diff
@@ -290,4 +290,25 @@ ruleTester.run('no-cycle', rule, {
       ],
     }),
   ),
+};
+
+ruleTester.run('no-cycle', rule, {
+  valid: [].concat(...cases.valid.map((testCase) => [testCase, {
```

Comment:
concat autoflattens arrays, so the spread isn't needed - which makes it much simpler and faster :-)
```suggestion
  valid: [].concat(cases.valid.map((testCase) => [testCase, {
```

Reply:
I tried without spread and the tests didn't work for me?

Reply:
oh hm, yeah i think it's trickier than that - keep the spreads for now.